imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -8,6 +8,7 @@
 
```

Comment:
I thought that filtering based on quality flags would be useful in the filter_good_data function. Is that something that you think would be useful. Just a thought.

Reply:
I think in level 1, I set the quality flags and it gets carried through. Then they want to use that in L3 to filter data based on these.